osi_hostvehicledata.proto

```diff
@@ -100,6 +100,37 @@ message HostVehicleData
         // Paragraph 42 of the German Road Traffic Admission Regulations (StVZO).
         //
         optional double curb_weight = 1;
+
```

Comment:
`curb_weight` is a static property of the vehicle, whereas `operating_status` is a dynamic property - doesn't mean they shouldn't go in the same message, but perhaps suggests they are different types of information and could be split out.  I guess the current naming / commenting of `VehicleBasics` is quite general so doesn't help us.

As above, no strong feelings, so happy to leave as-is, but mentioning in case someone else has similar ideas.